src/client/tile.cpp

```diff
@@ -1735,6 +1735,37 @@ bool TextureSource::generateImagePart(std::string part_of_name,
 			baseimg->drop();
 			baseimg = image;
 		}
+		/*
+			[opacity:R
+			Makes the base image transparent according to the given ratio.
+			R must be between 0 and 255.
+			0 means totally transparent.
+			255 means totally opaque.
+		*/
+		else if (str_starts_with(part_of_name, "[opacity:"))
+		{
```

Comment:
@nerzhul Thanks for the note. However, I used the style that the whole file uses. Could you tell me please, if I should
a) leave it as it is
b) only change it for the opacity filter
c) change it in the whole file